eng/common/pipelines/templates/steps/verify-links.yml

```diff
@@ -6,11 +6,15 @@ parameters:
   Recursive: $false
   CheckLinkGuidance: $true
   Urls: '(Get-ChildItem -Path ./ -Recurse -Include *.md)'
-  BranchReplaceRegex: "^(${env:SYSTEM_PULLREQUEST_SOURCEREPOSITORYURI}.*/(?:blob|tree)/)$(DefaultBranch)(/.*)$"
```

Comment:
You can still keep these here. The values will not get evaluated until the step that consumes them runs.

Reply:
Good to know. Will have it here and have a test.